detectors/node/opentelemetry-resource-detector-alibaba-cloud/package.json

```diff
@@ -10,7 +10,6 @@
     "compile": "tsc -p .",
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
-    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/resource-detector-alibaba-cloud --include-dependencies",
```

Comment:
note for reviewer: not necessary since `nx.json` has defined the task dependencies. Same applies for other packages